INTYGFV-16714: Update FK3226 texts

SamuelTilly opened the pull request.

ChrilleSigerhed reviewed (changes requested):
We also need to update the following:

- Text före val av formulär
- Om intyget-text
- Ny datumbegränsning i frågan “Till och med vilket datum” - Kalenderfunktionen tillåter endast val av datum från och med det datum som angetts i tidigare fråga “Ange när tillståndet blev akut livshotande” och framåt. Dvs datum innan datumet som angetts i frågan “Ange när tillståndet blev akut livshotande” är utgråade och inte klickbara. 

The last one might need some design, so we might wanna create a separate task for this.

SamuelTilly commented:
> We also need to update the following:
> 
> * Text före val av formulär
> * Om intyget-text
> * Ny datumbegränsning i frågan “Till och med vilket datum” - Kalenderfunktionen tillåter endast val av datum från och med det datum som angetts i tidigare fråga “Ange när tillståndet blev akut livshotande” och framåt. Dvs datum innan datumet som angetts i frågan “Ange när tillståndet blev akut livshotande” är utgråade och inte klickbara.
> 
> The last one might need some design, so we might wanna create a separate task for this.

Fixed the first two and i split the task into separate subtasks to handle the date issue.

ChrilleSigerhed reviewed (approved):
Nice!
